docs/content/tutorial/backups.md

```diff
@@ -123,6 +128,68 @@ kubectl apply -k kustomize/s3
 
 Watch your cluster: you will see that your backups and archives are now being stored in S3!
 
+### Using an AWS-integrated identity provider and role
+
+If you deploy PostgresClusters to AWS Elastic Kubernetes Service, you can take advantage of their 
+IAM role integration. When you attach a certain annotation to your PostgresCluster spec, AWS will 
+automatically mount an AWS token and other needed environment variables. These environment 
+variables will then be used by pgBackRest to assume the identity of a role that has permissions 
+to upload to an S3 repository.
+
+This method requires [additional setup in AWS IAM](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
+Follow the procedure there, but skip the third step:
+
+1\. Create an OIDC provider for your EKS cluster
+2\. Create an IAM policy for bucket access and an IAM role with a trust relationship with the 
+OIDC provider in step 1.
+
+The third step is to associate that IAM role with a ServiceAccount, but there's no need to 
+do that manually, as PGO does that for you. First, make a note of the IAM role's `ARN`.
+
+You can then make the following changes to the files in the `kustomize/s3` directory in the 
+[Postgres Operator examples](https://github.com/CrunchyData/postgres-operator-examples/fork) repository:
```

Comment:
:question: I'm thinking we wouldn't want the `fork` url here? Probably just
```suggestion
[Postgres Operator examples](https://github.com/CrunchyData/postgres-operator-examples) repository:
```

Reply:
I was conflicted about that, but almost all references to that repo include the `/fork` endpoint (in the context of "you'll want to start with this and then make changes"), so I left it in.

Reply:
Yes, hadn't noticed that detail. Thanks for the clarification. In that case, I agree with leaving it as is.